lib/protobuf/generators/field_generator.rb

```diff
@@ -20,7 +20,7 @@ class FieldGenerator < Base
       attr_reader :field_options
 
       def applicable_options
-        @applicable_options ||= field_options.map { |k, v| ":#{k} => #{v}" }
```

Comment:
You can't do `:.string` without ruby complaining, you have to do `:".string"`. Now that we are namespacing options and extensions in general, they will start with `.`